packages/@aws-cdk/aws-pipes-alpha/lib/pipeVariable.ts

```diff
@@ -0,0 +1,146 @@
+import { IResolvable, IResolveContext, captureStackTrace } from 'aws-cdk-lib';
+
+/** Reserved pipe variables
+* @see https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-pipes-input-transformation.html#input-transform-reserved
+*/
+export enum PipeVariable {
```

Comment:
Seems like we don't actually want this to be public.

Reply:
Ok so you suggest to get rid of the enum and just return the string values directly in the return value of for example `get pipeName ()` ?

Reply:
Yes

Reply:
Seems like this is redundant API surface for now.